### PR TITLE
Use the [[ operator not [

### PR DESCRIPTION
**Pull request by mrallen1**
Problem: `[` only works with numbers

### CODE REVIEW

File: .buildkite/scripts/make_image.sh [hunk 1]
```diff
@@ -16,7 +16,7 @@ docker build -t helium:$DOCKER_NAME -f "${DOCKERFILE_NAME}" .
 docker tag helium:$DOCKER_NAME "$MINER_REGISTRY_NAME:$DOCKER_NAME"
 docker push "$MINER_REGISTRY_NAME:$DOCKER_NAME"
 # only update latest if github tag contains `_GA$`
-if [ "$BUILDKITE_TAG" =~ _GA$ ]; then
+if [[ "$BUILDKITE_TAG" =~ _GA$ ]]; then
```
Comment (JayKickliter) [inline .buildkite/scripts/make_image.sh:19]: I had to look up this syntax. The trailing $ confused me until I saw that =~ was bash’s regex operator (so we’re looking for _GA at end of the line). 

I’ll approve this, but I almost think grep call makes sense here

**Review by JayKickliter — APPROVED** [review]

File: .buildkite/scripts/make_image.sh (hunk 1 above)
Comment (JayKickliter) [reply .buildkite/scripts/make_image.sh:19]: Actually, I’m even more confused now. The comment above implies that we’re searching for a literal dollar sign. Is that correct?

Comment (mrallen1) [reply .buildkite/scripts/make_image.sh:19]: I can add additional comment if you feel it would be more helpful

Comment (JayKickliter) [reply .buildkite/scripts/make_image.sh:19]: Yes please. Or at least correct the comment if we are not looking for a literal '$' but using it in the regex sense for end of line

File: .buildkite/scripts/make_image.sh [hunk 2]
```diff
@@ -16,7 +16,7 @@ docker build -t helium:$DOCKER_NAME -f "${DOCKERFILE_NAME}" .
 docker tag helium:$DOCKER_NAME "$MINER_REGISTRY_NAME:$DOCKER_NAME"
 docker push "$MINER_REGISTRY_NAME:$DOCKER_NAME"
 # only update latest if github tag contains `_GA$`
```
Comment (abhay) [inline .buildkite/scripts/make_image.sh:18]:
```suggestion
# only update latest if github tag contains `_GA`
```

**Review by vihu — APPROVED** [review]
lgtm

**Pull request merged**